experimental/rocm/tracing.c

```diff
@@ -37,8 +50,24 @@ struct iree_hal_rocm_tracing_context_t {
 
   // Event pool reused to capture tracing timestamps.
   hipEvent_t event_pool[IREE_HAL_ROCM_TRACING_DEFAULT_QUERY_CAPACITY];
+  
+  SmuDumpInitFunc f_smuDumpInit;
+  SmuDumpEndFunc f_smuDumpEnd;
+  SmuDumpOnceFunc f_smuDumpOnce;
+  RegDumpOnceFunc f_regDumpOnce;
+  SviDumpOnceFunc f_sviDumpOnce;
+  RegGetTraceRate f_regGetTraceRate;
+  SmuGetTraceRate f_smuGetTraceRate;
+  bool smutrace_enabled;
 };
 
+
+void iree_smutrace_callback(uint64_t did, const char* type ,const char* name, double value)
+{
+    IREE_TRACE_SET_PLOT_TYPE(name, IREE_TRACING_PLOT_TYPE_NUMBER, true, true, 0xFF0000FF);
```

Comment:
you only need to do the plot type set once after you confirm smutrace is loaded and used (whenever you call init)